pkl-core/src/main/java/org/pkl/core/project/ProjectPackager.java

```diff
@@ -366,6 +368,19 @@ private List<Path> collectPackageElements(Project project, Package pkg) {
     }
   }
 
+  private boolean isEmptyDirectory(Path path) {
```

Comment:
I think this is a good candidate for `IoUtils`.

Reply:
I found another solution.
See https://github.com/apple/pkl/pull/330/commits/a0ee462f59b654b1ccaf403c5132a24057734815#diff-debd0e8f0b916cc13d981b40ad3eaf74503c27fe814a26e3d28ef752acfa634aR341
I could get rid of this method at all.